desilike/observables/galaxy_clustering/correlation_function.py

```diff
@@ -153,12 +153,19 @@ def load_all(lmocks):
         return flatdata, list_y
 
     @plotting.plotter
-    def plot(self):
+    def plot(self, lax=None, kw_theo=None):
```

Comment:
I'd better pick `kw_theory` instead of `kw_theo`

Reply:
ok